bin/ExtractGName.R

```diff
@@ -1,5 +1,8 @@
+#!/usr/bin/env Rscript
+
 # Extract gene names as ordered by GO algo from ngs.plot output zip file.
 # Input should be the file name of the zip file without .zip suffix.
+library(stringr)
```

Comment:
-gene.list <- strsplit(gene.list[,1], ':')
-gname.list <- sapply(gene.list, function(x) x[1])

can be used in place of str_split_fixed

The stringr package just makes this simpler